flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/api/java/BatchTableEnvironment.java

```diff
@@ -109,24 +109,50 @@
 	<T> Table fromDataSet(DataSet<T> dataSet, String fields);
 
 	/**
-	 * Registers the given {@link DataSet} as table in the
-	 * {@link TableEnvironment}'s catalog.
-	 * Registered tables can be referenced in SQL queries.
+	 * Creates a view from the given {@link DataSet}.
+	 * Registered views can be referenced in SQL queries.
 	 *
-	 * The field names of the {@link Table} are automatically derived from the type of the{@link DataSet}.
+	 * <p>The field names of the {@link Table} are automatically derived
+	 * from the type of the {@link DataSet}.
+	 *
+	 * <p>The view is registered in the current catalog and database. To register the view in
```

Comment:
this comment can be a bit misleading as the view is registered actually only in the namespace of current catalog and db, but isn't physically located in the catalog/db, right?

Reply:
Will change to `The view is registered in the namespace of the current catalog and database.`